src/components/timing/TimingControls.vue

```diff
@@ -37,10 +38,11 @@ onBeforeMount(async () => {
 
 const handleStart = async () => {
   loadingStart.value = true
-  if (projectField.value !== '') {   
-    await dateStorage.postDailyHours( projectField.value, notesField.value )
-    await dateStorage.getDailyHours()
+  let start = new Date()
+  if (projectField.value !== '') {
+    await dateStorage.postDailyHours(projectField.value, notesField.value)
 
+    await dateStorage.getDailyHours(start.getFullYear(), start.getMonth() + 1, start.getDate())
```

Comment:
getDailyHours does not need the year, month and day anymore. dateStorage.getDailyHours() will be OK